client/js/lounge.js

```diff
@@ -918,6 +924,23 @@ $(function() {
 		location.reload();
 	}
 
+	function updateDesktopNotificationStatus(){
+		var checkbox = $("#desktopNotifications");
+		var warning = $("#warnDisabledDesktopNotifications");
+
+		if (Notification.permission === "denied"){
+			checkbox.attr("disabled", true);
+			checkbox.attr("checked", false);
```

Comment:
Checkboxes allow "indeterminate" state, like so: `checkbox.indeterminate = true;`, do you think it would be helpful to use it here, or when permission state is `default`?